<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="imglabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>欢迎使用烧录工具！！！！！！！</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Flash:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="flashPath">
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pathButton_1">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>300</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Partition:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="partitionPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="pathButton_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>串口选择：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="portComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>主控选择：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="boardComboBox">
       <item>
        <property name="text">
         <string>DFRduino UNO/Arduino Uno</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DFRduino Mega2560/Arduino Mega2560</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DFRobot Leonardo/Arduino Leonardo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DFRobot  Beetle Leonardo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DFRduino Nano</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FireBeetle-ESP32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FireBeetle-ESP8266</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Firebeetle M0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TinkerNode</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CurieNano </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bluno M3</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="informationEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="uploadButton">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>烧录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
